### PR TITLE
[feat]- Wishlist

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Add to Wishlist Functionality
- Update Number of products in Navbar
- Remove Products in Wishlist functionality.

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Good to merge

File: src/Components/Footer/footer.css [hunk 1]
```diff
@@ -5,6 +5,9 @@
         color: var(--text-color-2);
         padding: var(--size-11);
         line-height: 2.5;
+        /* position: absolute;
+        bottom: 0;
+        width: 100%; */
```
Comment (ashishkatiyar2404) [inline src/Components/Footer/footer.css:10]: you can remove comments

**Review by ashishkatiyar2404 — APPROVED** [review]
functionality of wishlist looks good to me .
You can merge

File: src/Components/Footer/footer.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply src/Components/Footer/footer.css:10]: Sure, Thankyou for pointing it out.

**Pull request merged**